.github/workflows/release.yaml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - '[0-9]+\.[0-9]+\.[0-9]+.*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
Should we start forcing `vX.Y.Z` tags? (this is forcing `X.Y.Z`)

Reply:
Oh yeah, that comment is incorrect it's from the template I copied it from but I edited the regex, we can use a prefix of `v` if you want -- I think we may need to update some code in wapm if we do though